resources/windows/wix/customactions.wxs.erb

```diff
@@ -50,6 +50,25 @@ End If
       ]]>
     </CustomAction>
 
+    <!-- Custom action to get the localised name for the "Everyone" (or World) user as
+         WixUtil:OSInfo Custom action doesn't return this value.
+         Equivalent of     (Get-WmiObject Win32_Account | where { $_.SID -eq "S-1-1-0" }).name
+      -->
+    <CustomAction
+      Id="SetEveryoneNameFromSid"
+      Script="vbscript"
+      Execute="firstSequence"
+      Return="ignore">
+        <![CDATA[
+      On Error Resume Next
+      Dim sid: sid = "S-1-1-0"
+      Dim objWMI : set objWMI = GetObject("winmgmts://./root\cimv2")
```

Comment:
Do the mixed slashes work OK?

Usually I see something like `winmgmts://./root/cimv2` - see https://msdn.microsoft.com/en-us/library/aa389292%28v=vs.85%29.aspx